wwwroot/config.json

```diff
@@ -85,6 +77,30 @@
       },
       "fallbackLanguage": "en"
     },
+    "searchBarModel": {},
+    "searchProviders": [
```

Comment:
```suggestion
    "searchProviders": [
      {
        "id": "search-provider/cesium-ion-search-provider",
        "type": "cesium-ion-search-provider",
        "url": "https://api.cesium.com/v1/geocode/search",
        "flightDurationSeconds": 1.5
      },
```

Reply:
added, just have to wait for terriajs release before merging this